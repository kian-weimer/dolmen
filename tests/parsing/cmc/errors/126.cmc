(define-system sys :input ((a Int)))