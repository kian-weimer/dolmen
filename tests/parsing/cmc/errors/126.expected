File "tests/parsing/cmc/errors/126.cmc", line 1, character 35-36:
1 | (define-system sys :input ((a Int)))
                                       ^
Error while parsing a system definition, read a closing parenthesis,
      but expected a subsystem declaration or function definition (:init,
      :trans, :inv).
